hack/origin-tests-common.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+. $(dirname "$0")/common.sh
```

Comment:
I think it's better to keep the sourcing of common explicit in each script
it also allow for the hypothetical use of the function here, without the common.sh if ever needed

Reply:
but we need OCP_VERSION for ORIGIN_TESTS_IMAGE (line 28) which is set in common.sh